src/containers/TreeViewer.tsx

```diff
@@ -0,0 +1,315 @@
+import React, { Component } from 'react';
```

Comment:
Would `AssetViewer` or `AssetTreeViewer` be a more apt name since we will have an equivalent one for relationships?

Reply:
good point, making change now :)